pennylane/operation.py

```diff
@@ -481,66 +481,71 @@ def matrix(self, wire_order=None):
         return expand_matrix(canonical_matrix, wires=self.wires, wire_order=wire_order)
 
     @classmethod
-    def _eigvals(cls, *params):
+    def compute_eigvals(cls, *params, **hyperparams):
         """Eigenvalues of the operator.
```

Comment:
Question: I did this inconsistently so far, but would like to call this the "canonical" eigenvalues, and explain that these are the eigenvalues wrt the canonical matrix. To me this was never clear, but it may just be me?

In this case, is there a clear path to and merit in adding `wires, wire_order` arguments to `eigvals` below? Since there is no order to them anyways? I don't think so myself...